common/install_mnist.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -ex
+
+mkdir -p /usr/local/mnist/
+
+cd /usr/local/mnist
+
+for img in train-images-idx3-ubyte.gz train-labels-idx1-ubyte.gz t10k-images-idx3-ubyte.gz t10k-labels-idx1-ubyte.gz; do
+  wget -q http://yann.lecun.com/exdb/mnist/$img
```

Comment:
Question: how do we know these downloads would succeed?

Reply:
Wget would fail with non-zero exit code if it wouldn't. But I can quickly add the test, that filesize is not zero